src/rgw/rgw_admin.cc

```diff
@@ -4247,10 +4247,26 @@ int main(int argc, const char **argv)
     bool need_cache = readonly_ops_list.find(opt_cmd) == readonly_ops_list.end();
     bool need_gc = (gc_ops_list.find(opt_cmd) != gc_ops_list.end()) && !bypass_gc;
 
+    std::string rgw_store = "rados";
```

Comment:
just want to confirm... is rados is a store? I was assuming that we have only two stores as of today... dbstore and we are adding motrstore. Please confirm

Reply:
lets keep as rados or motr only, as same is followed for radosgw which merged to ceph repo